examples/sara_work_area/OOP_Contact_Mgmt_System/genzsprthree_6_main.py

```diff
@@ -0,0 +1,104 @@
+from csv_contact_book import CsvContactBook
+from json_contact_book import JsonContactBook
+from CRUD_record import CRUDRecord
+from contact_record import ContactRecord
+from app_user import AppUser
+
+APP_USERS_FILE_PATH = "./app_usernames.csv"
+
+
+def main():
+    requested_format = "csv"
+    crud_record = None
+    app_running = True
+    logged_in = False
+    while app_running:
+        if logged_in:
```

Comment:
If we use not logged_in condition, it might make the code easy to understand because it seems like crud_record is used before its instantiated.